app/Enum/DonwloadVariantType.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+namespace App\Enum;
+
+/**
+ * Enum SizeVariantType.
+ */
+enum DonwloadVariantType: string
```

Comment:
```suggestion
/**
 * Enum DownloadVariantType.
 */
enum DownloadVariantType: string
```

Please also check the other comments. Most are not correct

Reply:
done. [5a864a8](https://github.com/LycheeOrg/Lychee/pull/1618/commits/5a864a8e7e2db244bc2f007cbe3462710ce8a12f)